hazelcast-sql/src/test/java/com/hazelcast/jet/sql/impl/connector/map/index/SqlIndexResolutionTest.java

```diff
@@ -228,74 +220,70 @@ private static int findFieldOrdinal(PartitionedMapTable table, String fieldName)
     }
 
     private void checkIndexUsage(
-            IMap<?, ?> map,
-            ExpressionType<?> f1,
-            ExpressionType<?> f2,
-            boolean firstResolved,
-            boolean secondResolved
+        IMap<?, ?> map,
+        ExpressionType<?> f1,
+        ExpressionType<?> f2,
+        boolean firstResolved,
+        boolean secondResolved
     ) {
         String field1Literal = toLiteral(f1, f1.valueFrom());
         String field2Literal = toLiteral(f2, f2.valueFrom());
 
         // The second component could be used if both components are resolved.
-        Usage expectedUsage;
```

Comment:
Would not it be worth keeping the original logic?

Reply:
Reworked test, thanks!